lib/sidekiq/scheduler.rb

```diff
@@ -137,7 +137,9 @@ def self.handle_errors
     def self.enqueue_job(job_config)
       config = job_config.dup
 
-      config['class'] = config['class'].constantize if config['class'].is_a?(String)
+      if config['class'].is_a?(String) && Object.const_defined?(config['class'])
+        config['class'] = config['class'].constantize
```

Comment:
`#constantize` is part of ActiveSupport API, so `config['class'].constantize` will fail on applications/projects which are not relying on ActiveSupport. It seems that `Kernel#const_get` would do the job.